jekyll/_cci2/overview.md

```diff
@@ -27,36 +27,9 @@ There are three basic ways to install CircleCI in your environment (AWS is curre
 2. A [clustered installation]({{ site.baseurl }}/2.0/aws/), suitable for production use for most teams
 3. A [highly-available configuration]({{ site.baseurl }}/2.0/high-availability/) (if supported by your license) to meet higher uptime requirements
 
-
-## Customer Use Cases 
-
-**Coinbase** runs CircleCI behind their firewall for security and reliability. The [Coinbase case study report](https://circleci.com/customers/coinbase/) reveals that using CircleCI reduced their average time from merge to deploy in half, reduced operations time spent on continuous integration maintenance from 50% of one person's time to less than one hour per week, and increased developer throughput by 20%. 
-
-**Fanatics** increased their team efficiency by 300% with CircleCI. See the [Fanatics case study report](https://circleci.com/customers/fanatics/).
-
-CircleCI enables **Cruise Automation** (a subsidiary of GM) to run many more simulations before putting code in a road test, see the [Cruise case study report](https://circleci.com/customers/cruise/).
-
-Refer to the [CircleCI Customers page](https://circleci.com/customers/) for the complete set of case studies for companies large and small who are using CircleCI.
-
-## Basic Features
-
-After a software repository in GitHub or GitHub Enterprise is added as a project to the CircleCI application, every new commit triggers a build and notification of success or failure through webhooks with integrations for Slack, HipChat, Campfire, Flowdock, or IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
-
-CircleCI may also be configured to deploy code to all major platforms, including the following:
-
-- AWS CodeDeploy
-- AWS EC2 Container Service (ECS)
-- AWS S3
-- Google Container Engine (GKE)
-- Heroku
-
-Other cloud service deployments can be easily scripted using SSH or by installing the API client of the service with your job configuration.
-
-Contact us at [Support](https://support.circleci.com/hc/en-us) to request a trial license.
-
 ## Build Environments
 
-By default, CircleCI 2.0 Builder instances automatically provision containers according to the image configured for each job in your `.circleci/config.yml` file. 
+By default, CircleCI 2.0 Builder instances automatically provision containers according to the image configured for each job in your `.circleci/config.yml` file. Refer to the [Introduction to Nomad Cluster Operation]({{ site.baseurl }}/2.0/nomad/) to learn more about the scheduler and how to perfom basic client and cluster operations.
```

Comment:
this seems a little confusing..... talking about 'the scheduler' when we have never mentioned anything called a scheduler previously. i feel like there is a simple way to mention here how we use nomad on 2.0 without just linking to our nomad documentation page..... maybe just pull a couple lines from https://circleci.com/docs/2.0/nomad & paste them here, along with the link ?

Reply:
ok, thanks! I added a line about 'job scheduler' to make this more clear.